### PR TITLE
build: test files and stories removed from shipping

### PR DESCRIPTION
**Pull request by theJohnnyMe**
**Describe pull-request**  
Removing test and story files from shipping in npm package as users do not need them, plus it reduces the size of the package.

**How to test**  
1. Open branch on local
2. Delete "dist" folder under packages/core
3. Run build command
4. Run npm pack
5. Check the size of package, it should be 3.6MB
6. Check "dist" folder and see if you can see any of test or story files in.
7. Do the same but on develop branch, size should be 4.1MB

### CODE REVIEW

File: packages/core/src/components/dropdown/test/filter/disabled/dropdown.e2e.ts-snapshots/tds-dropdown-filter-disabled-clicking-the-dropdown-icon-does-not-open-the-dropdown-list-1-linux.png
Comment (timrombergjakobsson) [inline packages/core/src/components/dropdown/test/filter/disabled/dropdown.e2e.ts-snapshots/tds-dropdown-filter-disabled-clicking-the-dropdown-icon-does-not-open-the-dropdown-list-1-linux.png:1]: How come this one has changed?

Comment (theJohnnyMe) [reply packages/core/src/components/dropdown/test/filter/disabled/dropdown.e2e.ts-snapshots/tds-dropdown-filter-disabled-clicking-the-dropdown-icon-does-not-open-the-dropdown-list-1-linux.png:1]: I think I miss the updates you have in your PR. I think once that one is merged this one will be OK, but worth double checking.

Comment (timrombergjakobsson) [reply packages/core/src/components/dropdown/test/filter/disabled/dropdown.e2e.ts-snapshots/tds-dropdown-filter-disabled-clicking-the-dropdown-icon-does-not-open-the-dropdown-list-1-linux.png:1]: Yes, most likely, I merged now btw!

**Review by timrombergjakobsson — APPROVED** [review]

**Review by nathalielindqvist — APPROVED** [review]
Nice!

**Pull request merged**